listenstore/listenstore/listenstore.py

```diff
@@ -207,7 +207,7 @@ def fetch_listens_for_range(self, uid, date_range, from_id, to_id, limit=None, o
         """
         query = """SELECT * FROM listens WHERE uid = %(uid)s AND """ + \
                 range_keys(len(date_range)) + \
-                """ AND id >= %(from_id)s AND id <= %(to_id)s
+                """ AND id > %(from_id)s AND id < %(to_id)s
```

Comment:
in our user profile view, we call `listenstore.fetch_listens`, so we should have the "one of min or max" check there too